Sources/Security/Signing+ResponseVerification.swift

```diff
@@ -95,3 +94,16 @@ extension HTTPResponse where Body == Data? {
     }
 
 }
+
+private extension HTTPRequest {
+
+    var requiresSignature: Bool {
+        guard self.path.supportsSignatureVerification else {
+            return false
+        }
+
+        let needsNonce = self.path.needsNonceForSigning
+        return !needsNonce || (needsNonce && self.nonce != nil)
```

Comment:
Hmm I wonder, is this part needed? In Android I'm only checking that `supportsSignatureVerification` is true and that entitlement verification is enabled. In that situation, we should always require a signature I think? Whether the path requires a nonce or not, shouldn't affect whether we require a signature I think.

Reply:
Oh the issue was some of the failing tests were wrong. I thought I needed this check but I don't! Fixed.